cdap-storage-spi/src/main/java/io/cdap/cdap/spi/data/StructuredTable.java

```diff
@@ -182,4 +182,12 @@ void increment(Collection<Field<?>> keys, String column, long amount)
    * @throws IOException if there is an error reading or deleting from the table
    */
   void deleteAll(Range keyRange) throws InvalidFieldException, IOException;
+
+/**
```

Comment:
Wrong alignment. Should be two spaces, not one.

Reply:
fixed